### PR TITLE
Scaffold for Statistics block on Landing pages

### PR DESCRIPTION
**Pull request by leenagupte**
⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

## What

Adds a statistics block to display CSV data on landing pages

## Why

[Trello card](https://trello.com/c/gvLYQCUp)

## Screenshots
Screenshot 2024-10-21 at 12 09 16

Rendered version: https://govuk-frontend-app-pr-4261.herokuapp.com/landing-page

### CODE REVIEW

File: app/models/block/statistics.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      data["y_axis_label"]
+    end
+
+    def rows
+      CSV.read(csv_file_path, headers: true).map(&:to_h)
+    end
+
+    def data_source_link
+      data["data_source_link"]
+    end
```
Comment (KludgeKML) [inline app/models/block/statistics.rb]: Do these need to be settable? Could they be read atttributes set in the initializer?

File: app/models/block/statistics.rb [hunk 2]
```diff
@@ -16,6 +16,10 @@ def y_axis_label
       data["y_axis_label"]
     end
 
+    def hide_legend
+      data["hide_legend"]
+    end
+
```
Comment (KludgeKML) [inline app/models/block/statistics.rb:22]: Same as last comment (could this be a read attribute set in the initializer)

**Comment by leenagupte** [comment]
@KludgeKML Thanks for your review. I've removed the unnecessary modelling and tidied up the commit history.

**Review by KludgeKML — APPROVED** [review]
LGTM

**Pull request merged**